CorsixTH/Lua/entities/humanoids/staff.lua

```diff
@@ -97,8 +97,21 @@ function Staff:tickDay()
    self:changeAttribute("happiness", 0.05)
   end
 
-  --TODO windows in your work space and a large space to work in add to happiness
-  -- working in a small space makes you unhappy
+  local room = self:getRoom()
+  -- It always makes you happy to see the outdoors (or windows to anywhere)
+  if room then
+    local count = room:countWindows()
+    self:changeAttribute("happiness", 0.001 * count)
```

Comment:
We might want to fiddle with this. Seems like too much advantage to scale linear for something that is free / cosmetic to me?